tests/disaster-recovery/regional-dr/test_failover.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+from time import sleep
+
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import wait_for_nodes_status, get_node_objs
+
+logger = logging.getLogger(__name__)
+
+
+class TestFailover:
+    def test_failover_when_primary_site_up(self, rdr_workload):
+        """
+        Test Failover action when primary site is UP
+
+        """
+        logger.info("Waiting 10 minutes before initiating Failover")
```

Comment:
why do we need to wait when primary and secondary cluster are up ??

Reply:
This was discussed in weekly meeting that we should wait for some time for IOs to run. Will modify the logging message to make it more clear.